### DISCUSSION TITLE
What countries held international elections in May?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#841] [Galaxy Trivia] [answered]
*answer with country emoji*

**Comment by hasanerzi** [comment, marked as answer]
🇨🇱   => 7 May: Chile, Constitutional Council
🇲🇷  => 13 May: Mauritania, Parliament  - 27 May: Mauritania, Parliament (2nd round)
🇹🇭   => 14 May: Thailand, House of Representatives
🇹🇷   => 14 May: Turkey, President (1st round) and Parliament - 28 May: Turkey, President (2nd round)
🇹🇱   => 21 May: East Timor, Parliament
🇬🇷   => 21 May: Greece, Parliament

**Comment by utkarsh006** [comment]
UK, India, Philippines, South Africa, Australia, France.
